src/plugins/intel_gpu/tests/unit/fusions/reduce_fusion_test.cpp

```diff
@@ -34,9 +34,13 @@ struct reduce_test_params {
 
 class ReduceFusingTest : public ::BaseFusingTest<reduce_test_params> {
 public:
-    void execute(reduce_test_params& p, bool is_dynamic = false) {
+    void execute(reduce_test_params& p, bool is_dynamic = false, bool use_default_input = false) {
```

Comment:
what about `use_planar_input` instead?

Reply:
How about adding comment why use default format for using planar input here?
4 or 5 dims test-cases could be added later.

Reply:
Updated.